src/platforms/apple/guides/ios/index.mdx

```diff
@@ -8,7 +8,7 @@ categories:
 
 <PlatformContent includePath="getting-started-install" />
 
-For other installation methods, please see our [documentation](https://docs.sentry.ip/platforms/apple/install/).
+For other installation methods, please see our [documentation](https://docs.sentry.io/platforms/apple/install/).
```

Comment:
```suggestion
For other installation methods, please see our [documentation](/platforms/apple/install/).
```

We don't normally include the domain on internal links.

Among other things, makes it easier to verify when PR branch is deployed to vercel.

Reply:
also you should just use "install/" as the path here as it's relative